### PR TITLE
Add namespace and RBAC needed for ingress-to-route

### PR DESCRIPTION
**Pull request by attiss**
We need to add RBAC configuration for ingress-to-route to work correctly on RedHat OpenShift on IBM Cloud 4.11 clusters.

---

Related cluster-openshift-controller-manager-operator PR: https://github.com/openshift/cluster-openshift-controller-manager-operator/pull/254/files

### CODE REVIEW

File: assets/cluster-bootstrap/route-controller-ns.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
```
Comment (csrwng) [inline assets/cluster-bootstrap/route-controller-ns.yaml:1]: Please rename this file to something like 00-route-controller-ns.yaml so it gets applied before the rolebinding

Comment (attiss) [reply assets/cluster-bootstrap/route-controller-ns.yaml]: Renamed, thank you!

**Review by rtheis — APPROVED** [review]
/lgtm
/approved

**Pull request merged**